src/main/resources/org/chainoptim/desktop/features/client/ClientOrdersView.fxml

```diff
@@ -4,23 +4,23 @@
 <?import javafx.scene.control.Label?>
 <?import javafx.scene.control.TableView?>
 <?import javafx.scene.control.TableColumn?>
+<?import javafx.scene.control.ScrollPane?>
 <AnchorPane xmlns="http://javafx.com/javafx"
             xmlns:fx="http://javafx.com/fxml"
-            fx:controller="org.chainoptim.desktop.features.client.controller.ClientOrdersController"
-            prefHeight="400.0" prefWidth="600.0">
+            fx:controller="org.chainoptim.desktop.features.client.controller.ClientOrdersController">
 
-    <Label fx:id="clientName" />
-    <TableView fx:id="tableView" AnchorPane.topAnchor="20.0" AnchorPane.leftAnchor="0.0" AnchorPane.rightAnchor="0.0" AnchorPane.bottomAnchor="0.0" styleClass="table-view">
-        <columns>
-        <TableColumn text="Order ID" fx:id="orderIdColumn"/>
-        <TableColumn text="Client ID" fx:id="clientIdColumn"/>
-        <TableColumn text="Product ID" fx:id="productIdColumn"/>
-        <TableColumn text="Quantity" fx:id="quantityColumn"/>
-        <TableColumn text="Status" fx:id="statusColumn"/>
-        <TableColumn text="Order Date" fx:id="orderDateColumn"/>
-        <TableColumn text="Estimated Delivery Date" fx:id="estimatedDeliveryDateColumn"/>
-        <TableColumn text="Delivery Date" fx:id="deliveryDateColumn"/>
-
-        </columns>
-    </TableView>
+    <ScrollPane fx:id="scrollPane" styleClass="scroll-pane" fitToWidth="true" AnchorPane.topAnchor="0.0" AnchorPane.leftAnchor="0.0" AnchorPane.rightAnchor="0.0" AnchorPane.bottomAnchor="40.0">
```

Comment:
Decide on either StackPane and ScrollPane for consistency. Also not sure if that outer AnchorPane is necessary.